composer.json

```diff
@@ -17,7 +17,7 @@
         "php": "^5.3.6 || >=7.0 <7.2",
         "ext-tokenizer": "*",
         "sebastian/diff": "^1.1",
-        "symfony/console": "^2.3 || ^3.0",
+        "symfony/console": "^2.4 || ^3.0",
```

Comment:
As I do always answer to you @sstok, if we could work properly on both 2.8 and 2.4 no need for dropping 2.4.
I saw a lot of legacy, monolithic projects that updates their codebase to have better CS and this kind of change would disallow them to install the tool as dev-dependency and require them to install it as phar file.